requirements.txt

```diff
@@ -1,6 +1,7 @@
 addict
 cdsapi
 dask
+icoscp
```

Comment:
I think `folium` also needs to be added here as I got a ModuleNotFound error after refreshing the environment using pip (`pip install -r requirements.txt`).

Reply:
Ah yes, I might add an issue to their repo, I had the same problem.